### PR TITLE
[V360-8810] feat testing : Breadcrumbs test cases

### PR DESCRIPTION
**Pull request by NamrataAjmeri**
## Scope
- [x] Enhancement: backwards-compatible functionality added
- [ ] Bug-Fix / Patch: backwards-compatible bug fix / revision
- [ ] RFC: request for comments; discussion only

## Context
Added unit test case to increase code coverage.

## Summary of Changes
Wrote unit test cases for Breadcrumbs.tsx file using React testing library.

## Pre-Merge Checklist
- [ ] Changelog entry? (Summary in `/changelog/<section>/<issue_number>.md`)
- [ ] Linked to [Jira issue](https://support.atlassian.com/bitbucket-cloud/docs/use-smart-commits/) ?
- [ ] Labels tagged?

### CODE REVIEW

File: src/Breadcrumbs/Breadcrumbs.test.js [hunk 1]
```diff
@@ -1,77 +1,81 @@
...
-    );
-
-    expect(wrapper.find(".breadcrumb__item").text()).toEqual(title);
-
-    // If only 1 breadcrumb, not clickable (current page open)
-    wrapper.find(".breadcrumb__item").simulate("click");
-    expect(onClick).not.toHaveBeenCalled();
+    expect(container.firstChild.className).toMatch(/common__breadcrumbs/);
+    expect(container.firstChild.className).toMatch(/TestClassName/);
+    expect(container.firstChild.style.background).toMatch(/red/);
```
Comment (johnisr) [inline src/Breadcrumbs/Breadcrumbs.test.js:26]: Still undecided on whether we should test implementation details with react testing library, maybe ask Chung about it but I guess we can leave this as is for now

File: src/Breadcrumbs/Breadcrumbs.test.js [hunk 2]
```diff
@@ -1,77 +1,81 @@
...
     ];
+    const { getByText, container } = render(
+      <Breadcrumbs
+        breadcrumbs={breadcrumbs}
+        className={className}
+        style={style}
+      />
+    );
+    const firstBreadcrumbElement = getByText("test");
+    const secondBreadcrumbElement = getByText("test2");
```
Comment (johnisr) [inline src/Breadcrumbs/Breadcrumbs.test.js:59]: Adding `aria-label="breadcrumbs"` and `aria-current="location"` as described [here](https://www.aditus.io/patterns/breadcrumbs/) may allow us to use getByRole or getByLabelText instead.

**Review by johnisr — COMMENTED** [review]
One extra thing that would be good is to see if you can test breadcrumbs while using the `<Page>` component. That's really the only time we'll be using breadcrumbs in the other repos. Goes with the whole "The more your tests resemble the way your software is used..."

File: src/Breadcrumbs/Breadcrumbs.test.js (hunk 2 above)
Comment (NamrataAjmeri) [reply src/Breadcrumbs/Breadcrumbs.test.js:59]: Yes I added role attribute to access it by getByRole.

File: src/Breadcrumbs/Breadcrumbs.test.js (hunk 1 above)
Comment (NamrataAjmeri) [reply src/Breadcrumbs/Breadcrumbs.test.js]: This will be addresses when I write test cases for Page component.

**Review by johnisr — APPROVED** [review]

**Pull request merged**